### PR TITLE
feat: add more arabic names dataset

### PR DESCRIPTION
**Pull request by wael-Fadlallah**

### CODE REVIEW

**Review by import-brain — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]
~~Formatting looks good.~~ I cannot verify the actual translations though.

**Review by JessicaSachs — APPROVED** [review]
Yeah we can't validate the translations except for using Google Translate.

**Comment by Shinigami92** [comment]
> Yeah we can't validate the translations except for using Google Translate.

@JessicaSachs Was it expected to merge something into main that was targeted for v6.2?
We currently highly try only to focus on absolutely needed project stability changes.

File: src/locales/ar/name/female_first_name.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default [
+ 'فاطمه',
```
Comment (Shinigami92) [inline src/locales/ar/name/female_first_name.ts:2]: This and the other files are not formatted at all and now causing CI errors. We need 3 maintainers to fix the main branch. 1 that creates the PR and two approvals.
Until then we cannot rebase any of the other PRs anymore 😬

**Comment by wael-Fadlallah** [comment]
@Shinigami92 This could be considered a stability change, the reason I add these files is when I used AR locale the findName Method returns an English first name with an Arabic last name, I tried to debug the actual issue but I couldn't figure it out but adding these files returns a valid Arabic name

**Comment by Shinigami92** [comment]
> @Shinigami92 This could be considered a stability change, the reason I add these files is when I used AR locale the findName Method returns an English first name with an Arabic last name, I tried to debug the actual issue but I couldn't figure it out but adding these files returns a valid Arabic name 

I know, sorry if we did not clarify that to much right now, but we maintainers are currently working daily multiple hours on rewriting e.g. the whole test suite and introduce linting and such.
Adding a new locale is definitely planned and wanted. But not right now. 

Internally we plan what we are currently focus on, and more and more code on top of what we currently need to fix and stabilize is currently not helping.

Sorry, I already work like 10h per day currently on the Code base. I can really do not more right now to speedup the process.

**Comment by ST-DDT** [comment]
@wael-Fadlallah Could you please create a new PR with your changes?
Your contributions are appreciated and we are sorry for the trouble/additional work we caused you.

**Comment by wael-Fadlallah** [comment]
@ST-DDT No need for apologies I'm happy to help, just to make sure everything will go smoothly this time can I know what is the formatting issue that caused troubles in my first PR

**Comment by Shinigami92** [comment]
Yeah for sure, I were wrongly indented lines and the trailing `+`-es

You don't need to care for these yourself, just run `pnpm run format` or do not bypass the lint-stage that should prevent such commits

**Pull request merged**